package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Bloomberg Open API HTTP Proxy",
   "main": "server.js",
   "dependencies": {
-    "blpapi": "^1.2.0",
+    "blpapi": "thmiceli/blpapi-node#move-wrapper-to-node",
```

Comment:
Until the corresponding PR for blpapi-node has been merged we will need this for CI to succeed.